whelktool/scripts/syslib/ISNI.groovy

```diff
@@ -0,0 +1,43 @@
+PrintWriter failedAuthIDs = getReportWriter("failed-authIDs")
+PrintWriter scheduledForUpdating = getReportWriter("scheduled-updates")
```

Comment:
We also used to do it this way (and there's nothing wrong with it!), but since a few months, this is now also done implicitly.

So there will be (other) logfiles corresponding to scheduled-to-save and failed-to-save, even if you don't do this. Might be good to know!

Reply:
Yes, I've seen this (other log-files). That these remain in this script is because we started on it quite some time ago, and I simply forgot to remove them at a later point.